pom.xml

```diff
@@ -97,10 +99,20 @@
         <artifactId>org.eclipse.jgit</artifactId>
         <version>${jgit.version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.awaitility</groupId>
+        <artifactId>awaitility</artifactId>
+        <version>${awaitility.version}</version>
+      </dependency>
       <dependency>
         <groupId>org.zeroturnaround</groupId>
         <artifactId>zt-exec</artifactId>
-        <version>1.11</version>
+        <version>${zt-exec.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.yaml</groupId>
```

Comment:
Same here, I could not see any places where you used any class of this package. Is this really required?